lib/cocoapods/resolver.rb

```diff
@@ -543,8 +535,10 @@ def handle_resolver_error(error)
     #
     # @return [Bool]
     def spec_is_platform_compatible?(dependency_graph, dependency, spec)
+      return true if locked_dependencies.vertex_named(spec.name) # HACK: this probably isn't safe?
```

Comment:
what about the comment here?

Reply:
just making sure this is covered before merging.

Reply:
I _think_ it should be safe, after thinking it over, since if there are other changes then the spec won't be in `locked_dependencies` ?